app/pages/History/TransactionDetails.view/TransactionDetails.view.tsx

```diff
@@ -165,25 +164,8 @@ const TransactionDetails: FC<TransactionDetailsViewProps> = ({
         <CardContent>
           {outputTxoIds.map((txoId) =>
             direction === 'tx_direction_sent'
-              ? renderRow(
-                  txoId,
-                  <TransactionInfoLabel
-                    valuePmob={txos.txoMap[txoId].valuePmob}
-                    sign={sign}
-                    label=" MOB"
-                  />,
-                  true
-                )
-              : renderTXODetailsRow(
-                  txoId,
-                  <TransactionInfoLabel
-                    valuePmob={txos.txoMap[txoId].valuePmob}
-                    sign={sign}
-                    label=" MOB"
-                  />,
-                  txoValidations[txoId],
-                  true
-                )
+              ? renderRow(`ID: ${txoId}`, undefined, true)
```

Comment:
the value of the transaction is already being rendered above on line 155. Removing the transaction value from this section allows us to render the entire transaction ID, and gets around some concurrency issues resulting from getting the same value from two different locations.